api/service/v1alpha1/service.proto

```diff
@@ -385,25 +385,20 @@ message DeletePromotionPolicyResponse {
   /* explicitly empty */
 }
 
-message Project {
-  string name = 1;
-  google.protobuf.Timestamp create_time = 2;
-}
```

Comment:
This moved to `types.proto` where we have the protobuf definitions of anything that's also a CRD.